cadc-util/src/main/java/org/opencadc/persist/Entity.java

```diff
@@ -91,7 +91,12 @@
 import org.apache.log4j.Logger;
 
 /**
- * Base class for entity persistence.
+ * Base class for entity persistence. The metaChecksum algorithm implemented here has a
```

Comment:
The default to false is just in the deprecated ctors for backwards compatibiity - in case we did a rebuild of something from inventory or caom2 and it slipped past tests. 

It's deprecated so new usage is encouraged to use the explicit ctor(s) instead.

Reply:
I understand that, but I think is better to document the bug free functionality of the class  (name digest on) and mention the consequences of the flag being turned off.